bower.json

```diff
@@ -1,24 +0,0 @@
-{
-  "name": "tota11y",
-  "version": "0.1.2",
-  "description": "An accessibility visualization toolkit",
-  "main": "build/tota11y.js",
-  "authors": [
-    "Jordan Scales <scalesjordan@gmail.com>"
-  ],
-  "license": "MIT",
-  "keywords": [
-    "tota11y",
-    "accessibility",
-    "khan"
-  ],
-  "homepage": "http://khan.github.io/tota11y/",
-  "moduleType": [],
-  "ignore": [
-    "**/.*",
-    "node_modules",
-    "bower_components",
-    "test",
-    "tests"
-  ]
-}
```

Comment:
bower is dead and I think keeping this around is only going to leave it going stale